WDAC-Policy-Wizard/app/src/CustomRuleConditionsPanel.cs

```diff
@@ -677,6 +693,18 @@ private void Button_Browse_Click(object sender, EventArgs e)
                         leafCertSubjectName = cert.SubjectName.Name;
                         leafCertSubjectName = FormatSubjectName(leafCertSubjectName);
 
+                        // Add all the EKUs from the leaf cert. Filter out the code signing one on the UI
```

Comment:
why filter out something you'll just add a duplicate of?

Would it not make more sense to have check that the signer has the code signing EKU here and bail if not?